gen_cert.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
 
         def interstitial_factory():
             """ Generate default values for interstitial_texts defaultdict """
-            return itertools.repeat(cert_data.get('interstitial', {}).get('Pass', '')).next
+            return itertools.repeat(cert_data.get('interstitial', {}).get('Pass', '').encode('utf-8')).next
```

Comment:
not necessary as the "encode" already does that...

This encode line makes sure that any value given not listed in the yaml is encoded utf-8